blazor/speeddial/display-modes.md

```diff
@@ -41,8 +90,76 @@ The `Linear` directions of Speed Dial are as follows:
 
 ```
 
-![Blazor Speed Dial Linear Mode](./images/Blazor-SpeedDial-LinearMode.png)
+![Blazor SpeedDial component with linear mode](./images/Blazor-SpeedDial-LinearMode.png)
+
+## Radial mode 
+
+The action items in Blazor Speed Dial can be displayed in a circular patter like a radial menu by setting [Mode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_Mode) property. You can customize the `Direction`, `StartAngle`, `EndAngle` and `Offset` by setting [SpeedDialRadialSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SpeedDialRadialSettings.html) tag directive.
```

Comment:
Check spell error circular patter => circular pattern

Reply:
Corrected the spelling error